xl_auth/grant/views.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""Grant views."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from flask import Blueprint, abort, flash, redirect, render_template, url_for
+from flask_babel import lazy_gettext as _
+from flask_login import current_user, login_required
+
+from .models import Grant
+
+blueprint = Blueprint('grant', __name__, url_prefix='/grants', static_folder='../static')
+
+
+@blueprint.route('/')
+@login_required
+def home():
+    """Grant landing page."""
+    if not current_user.is_admin:
+        abort(403)
+
+    grants = Grant.query.all()
+
+    return render_template('grants/home.html', grants=grants)
+
+
+@blueprint.route('/delete/<int:id>', methods=['GET', 'DELETE'])
+@login_required
+def delete(id):
+    """Delete grant."""
+    if not current_user.is_admin:
+        abort(403)
+
+    grant = Grant.query.get(id)
```

Comment:
Any reason for not using the "Grant.get_by_id" that we're using in testing? Just curious.. Does this behave differently somehow?

Reply:
Fixed.